genie-agent/src/main/java/com/netflix/genie/agent/execution/services/impl/JobSetupServiceImpl.java

```diff
@@ -547,6 +550,21 @@ String composeScript() {
 
             sb.append(NEWLINE);
 
+            sb
+                .append("# During setup, redirect stdout and stderr of this script to a log file").append(NEWLINE)
```

Comment:
I know we have some comments in the run script today. Do we want to keep them or just remove them and leave them as comments in the java code? It would save serialization/bytes/etc. Could also add some flag to the job request that would enable this eventually optionally. Just a thought.

Reply:
I like the fact that the script is self-contained and someone looking at it (not us) can understand what these obscure things are, e.g.:
```
exec 1>&6 6>&-
exec 2>&7 7>&-
```